build/Build.Steps.cs

```diff
@@ -440,7 +440,8 @@ void CopyNativeDependenciesToStore(AbsolutePath file, JsonDocument jsonDocument,
 
                 void RemoveDuplicatedLibraries(string depsJsonContent, IReadOnlyList<string> architectureStores)
                 {
-                    var duplicatedLibraries = new List<(string Name, string Version)> { (Name: "Microsoft.Extensions.Logging.Abstractions", Version: "6.0.0") };
+                    // TODO: Consider to remove this method
```

Comment:
Whole method is not needed now.
I expect, it will be needed in the future. I can remove it completely, but I would like to hear your recommendation.

Reply:
I will keep it 7c12e77edd9ff2626b021d55b7f9567716e7388a